src/Tribe/Views/V2/Assets.php

```diff
@@ -528,6 +528,20 @@ public function should_enqueue_frontend() {
 			return $this->should_enqueue_frontend;
 		}
 
+		// Detect plugin. For frontend only
+		include_once ABSPATH . 'wp-admin/includes/plugin.php';
+
+		/**
+		 * Checks whether the page is being viewed in Elementor preview mode.
+		 * 
+		 * @since TBD
+		 * 
+		 * @return bool $should_enqueue Should the frontend assets be enqueued.
+		 */
+		if ( is_plugin_active( 'elementor/elementor.php' ) && \Elementor\Plugin::$instance->preview->is_preview_mode() ) {
+			return $this->should_enqueue = true;
+		}
```

Comment:
We should use the same conditional as the code from the integrations. To determine if elementor is active.
```
		if ( ! defined( 'ELEMENTOR_PATH' ) || empty( ELEMENTOR_PATH ) ) {
			return;
		}
```

Reply:
@bordoni I've updated the conditional